src/test/common/clara/test_simple_rules.cljc

```diff
@@ -194,3 +194,14 @@
     (is (has-fact? (:cold @side-effect-holder) (->Temperature -10 "MCI") ))
 
     (is (has-fact? (:subzero @side-effect-holder) (->Temperature -10 "MCI")))))
```

Comment:
This doesn't really seem like a "simple rule" sanity test to me.  I suppose we could put it here, but IMO it fits better in test-dsl since it revolves around edge cases of ways to represent the logical relationships implied by rules, not edge cases of the logical relationships themselves.  Test-dsl is a cross-platform test file as well.